package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "type": "commonjs",
   "author": "Wes Todd <wes@wesleytodd.com>",
-  "license": "ISC",
+  "license": "MIT",
```

Comment:
@wesleytodd I assume the `LICENSE` file is correct here?

Reply:
Yep, I changed the license when I moved it but forgot this!  Thanks!